lib/commands/destroy.js

```diff
@@ -51,6 +54,10 @@ module.exports = Command.extend({
       commandOptions.pod = !commandOptions.pod;
     }
 
+    if (commandOptions.in) {
+      commandOptions.in = path.relative(this.project.root, commandOptions.in);
```

Comment:
This is still a relative path, can you tweak this to match the logic in generate?